test/core/crawlers/basic_crawler.test.ts

```diff
@@ -53,8 +53,11 @@ describe('BasicCrawler', () => {
         await localStorageEmulator.init();
     });
 
-    afterAll(async () => {
+    afterEach(async () => {
```

Comment:
Any reason for this change and the others like this? 👀

Reply:
Yes. The `HttpCrawler` tests fail otherwise because there are tests that use the same URLs

Reply:
Each test should be considered a separate environment

Reply:
but thats how it was supposed to be working already, `destroy` call just cleans up the files for all opened storages, this shouldn't be needed as `init` is what ensures you have distinct storages

(with that said, it shouldnt hurt either if we destroy them early, but it rings a bell if the change was required)

> Yes. The HttpCrawler tests fail otherwise because there are tests that use the same URLs

so you have tests that are rather wrong, but instead of changing them you changed how we handle this problem everywhere? :]

Reply:
> so you have tests that are rather wrong, but instead of changing them you changed how we handle this problem everywhere? :]

Why my tests would be rather wrong? I don't see how that would be. Anyway, I just tested again with `afterAll` and it seems to... pass? wtf is going on 🤔 I guess I'll revert the changes then

Reply:
I mean if your tests required this change, they were probably wrong, as this change should not solve what you were trying to solve :] Or the issue was elsewhere, if they pass now 🤷  :]

Reply:
No I haven't changed the tests nor the code I swear :joy: